codebuild/spec/buildspec_sanitizer.yml

```diff
@@ -105,9 +105,7 @@ phases:
           -DCMAKE_C_COMPILER=/usr/bin/$COMPILER \
           -DCMAKE_PREFIX_PATH=/usr/local/$S2N_LIBCRYPTO \
           -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-          -DASAN=ON
+          -DASAN=ON \
+          -DUBSAN=ON
       - cmake --build ./build -- -j $(nproc)
-  post_build:
-    on-failure: ABORT
-    commands:
```

Comment:
This is left over from your previous approach (separate builds). You don't need this change anymore.